.dev/scopes.txt

```diff
@@ -0,0 +1,8 @@
+model
+world
+object
+view
+controller
+cmake
+readme
+git
```

Comment:
I'd add another scope for the tasking system (i.e., everything related to running things in parallel)